### DISCUSSION TITLE
EACCES: permission denied, open...

### DISCUSSION THREAD

**Discussion by PatchworkPirate** [requarks/wiki#5323] [Help / Questions] [answered]
Hi,

I was messing around in portainer trying to update the network rules for my wikijs and accidentally created the new one using the same database. I quickly shut down the container and booted back up the original container, it seems like looking at the logs it's retained all of the wiki entries. Unfortunately when trying to access the wiki I get this:
![image](https://user-images.githubusercontent.com/76474964/170075419-fa51113f-2ac9-406b-978f-f62937d5e56c.png)

I've accessed the folder it's trying to look into and it seems to be empty? Any help would be massively appreciated.

**Comment by PatchworkPirate** [comment, marked as answer]
This has been resolved with a fresh wiki install :)

**Comment by melihkaragoz** [comment]
just add it in compose.yml file:

```
services:
    wiki:
        # Other configuration lines.
        user: "1001:1001"
```
        

If this solution couldn't fix your case, find out (explained below) which user is owner /wiki/data/cahce directory. Then change the user's name with "1001" above.
Try again.

### How to find out owner of that directory?

execute this line to enter wiki container's cmd line
`sudo docker exec -it wiki bash`
then,
`ls -la /wiki/data`

![tmp](https://github.com/requarks/wiki/assets/34073531/cb149aa0-4c72-41d0-8f1c-cd7e9a67a3f8)



it worked for me.
